draft-ietf-quic-http.md

```diff
@@ -782,7 +758,86 @@ HTTP_MULTIPLE_SETTINGS (0x10):
 HTTP_RST_CONTROL_STREAM (0x11):
 : A message control stream closed abruptly.
 
-## Mapping HTTP/2 Error Codes
+
+# Considerations for Transitioning from HTTP/2
+
+HTTP/QUIC is strongly informed by HTTP/2, and bears many similarities.  This
+section points out useful differences from HTTP/2 and describes how to map
+HTTP/2 extensions into HTTP/QUIC.
+
+## HTTP Frame Types
+
+Many framing concepts from HTTP/2 can be elided away on QUIC, because the
+transport deals with them. Because frames are already on a stream, they can omit
+the stream number. Because frames do not block multiplexing (QUIC's multiplexing
+occurs below this layer), the support for variable-maximum-length packets can be
+removed. Because stream termination is handled by QUIC, an END_STREAM flag is
+not required.
+
+Frame payloads are largely drawn from {{!RFC7540}}. However, QUIC includes some
+features (e.g. flow control) which are also present in HTTP/2. In these cases,
+the HTTP mapping does not re-implement them. As a result, some frame types are
+not required in HTTP/QUIC. Where an HTTP/2-defined frame is no longer used, the
+frame ID has been reserved in order to maximize portability between HTTP/2 and
+HTTP/QUIC implementations. However, equivalent frames between the two mappings
+are not identical.
+
```

Comment:
Another important difference is that while there is an absolute ordering between every frame in h2, this is not true for frames on different streams in QUIC.  You mention this below, but only obliquely.  I think that it's important enough for a paragraph.